docs/cookbook/images/images-on-entity.rst

```diff
@@ -313,23 +313,20 @@ Register the form extension as a service:
             tags:
                 - { name: form.type_extension, extended_type: Sylius\Bundle\ShippingBundle\Form\Type\ShippingMethodType }
 
-11. Override the definition of the ImageUploader service
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+11. Override the definition of the ImagesUploadListener service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 In order to handle the image upload you need to attach the image upload listener to the ``ShippingMethod`` entity events:
 
 .. code-block:: yaml
 
     # services.yml
     services:
-        sylius.listener.image_upload:
+        app.listener.image_upload:
             class: Sylius\Bundle\CoreBundle\EventListener\ImagesUploadListener
+            decorates: sylius.listener.image_upload
             arguments: ['@sylius.image_uploader']
```

Comment:
Don't we need `@app.listener.image_upload.inner` here now?

Reply:
We're not using it, because we're basically abusing service decoration. :stuck_out_tongue: